.nvmrc

```
@@ -1 +1 @@
-14.15.1
+16.X
```

```suggestion
16.9.1
```
Looks like nvm requires explicit pinning to a version:

```
Found '/Users/radeksimko/gopath/src/github.com/hashicorp/vscode-terraform/.nvmrc' with version <16.X>
Version '16.X' not found - try `nvm ls-remote` to browse available versions.
```